docs_src/microservices/device/profile/Ch-DeviceProfile.md

```diff
@@ -1,6 +1,93 @@
 # Device Profile
 
-![image](EdgeX_DeviceProfile.png)
+The device profile describes a type of device within the EdgeX system. Each
+device managed by a device service has an association with a device profile,
+which defines that device type in terms of the operations which it supports.
+
+For a full list of device profile fields and their required values see the [Device Profile Reference](./Ch-DeviceProfileRef.md)
+
+Identification
+--------------
+
+The profile contains various identification information. The `Name` field is required and must be unique in an EdgeX deployment. Other fields are optional and provide additional details:
+
+* Description
+* Manufacturer
+* Model
+* Labels
+
+DeviceResources
+---------------
+
+A deviceResource specifies a value within a device that may be read from or
+written to either individually or as part of a deviceCommand. It has a name
+for identification and a description for informational purposes.
+
+The device service allows access to deviceResources via its `device`
+REST endpoint.
+
+The `Attributes` in a deviceResource are the device-service-specific parameters
+required to access the particular value. Each device service implementation
+will have its own set of named values that are required here, for example a
+BACnet device service may need an Object Identifier and a Property Identifier
+whereas a Bluetooth device service could use a UUID to identify a value.
+
+The `Properties` of a deviceResource describe the value. Each device resource
+is given two properties, `value` and `units`. The following fields are
```

Comment:
The 'value' property gives more detail about the value collected or set; such as the type of the data collected, whether the value can be read, written or both.  The 'units' property gives more detail about the unit of measure associated with the value.   The unit of measure can also have a type (although it is most often just a String), a default value, whether the unit can be read, written or both (although it is most often set to read only).